### DISCUSSION TITLE
Can this plugin be called with command-line arguments? And can the results of the call be returned to the command line?

### DISCUSSION THREAD

**Discussion by workcheng** [AutomaApp/automa#1292] [Q&A]
I have an application and I want to automate some tasks using this plugin, such as opening web pages, logging in, retrieving some current states, and returning the results to my application. Can I use this plugin to accomplish these tasks?

**Comment by flashvnn** [comment]
I think your application write a **restful api** with specific port, example **127.0.0.1:8080/api/automa**, after  **automate** complete it's task you use javascript block and use fetch function to send data back to your application.

**Reply by workcheng** [reply]
Yes, that's it.
